uwblueprint/social-diversity-for-children
# Class Registration Agreement Components

### Notion ticket link

[Class Registration Agreement Components](https://www.notion.so/uwblueprintexecs/Class-Registration-Agreement-Components-0c02fe731b4447f083289d14e12910c8)

### Implementation description

- Created 3 new components (one for each of the registration agreements that must be accepted to register for a SDC program)
- Each component has text fields, a checkbox, and a button (which is disabled until the checkbox is selected)
- Routing from one agreement to the next is achieved using the useRouter hook when the next button is clicked

### Steps to test
<img width="1439" alt="Screen Shot 2021-10-03 at 5 32 48 PM" src="https://user-images.githubusercontent.com/23081661/135772212-4b87d4eb-6bcf-4485-9740-c65d59038404.png">

<img width="1435" alt="Screen Shot 2021-10-03 at 5 33 15 PM" src="https://user-images.githubusercontent.com/23081661/135772214-3efd88d2-381e-41c9-82ca-e487b3b8734a.png">

<img width="1433" alt="Screen Shot 2021-10-03 at 5 33 24 PM" src="https://user-images.githubusercontent.com/23081661/135772217-a697b009-4c41-4e90-add6-9b6edfe5ed09.png">


1. Visit http://localhost:3000/en/media-release-form
2. Try clicking the "Next" button when the checkbox is NOT selected (nothing should happen)
3. Try clicking the "Next" button when the checkbox is selected (you should be taken to the "Participant Waiver" page)
4. Repeat step 2 on the Participant Waiver page
5. Repeat step 3 on the Participant Waiver page (you should be taken to the "Terms and Conditions" page)

Note: Clicking the "Next" button on the "Terms and Conditions" page does not redirect anywhere because it wasn't specified what the next page in the registration process should be.

### What should reviewers focus on?

- Does the form content match the Figma (i.e. no missing words, spelling mistakes, etc.)?
- Does the design match the Figma?
- Are you able to move from page-to-page when clicking the 'Next' buttons?

### Checklist
